src/main/java/frc/robot/subsystems/CoordsOutputSubsystem.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.json.simple.JSONObject;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class CoordsOutputSubsystem extends SubsystemBase{
+
+    private JSONObject jsonOutput;
+
+    public Swerve s_Swerve;
+
+    private static CoordsOutputSubsystem s_CoordsOutputSubsystem;
+
+    public static CoordsOutputSubsystem getInstance() {
+        if (s_CoordsOutputSubsystem == null) {
+            s_CoordsOutputSubsystem = new CoordsOutputSubsystem();
+        }
+        return s_CoordsOutputSubsystem;
+    }
+
+
+
+    public CoordsOutputSubsystem() {
+
+        s_Swerve = Swerve.getInstance();
+        jsonOutput = new JSONObject();
+
+    }
+
+    public void appendJSON(int id, String value) {
+
+        jsonOutput.put(id, value);
+
+    }
+
+    public void printJSON() {
+
+        try {
+            FileWriter file = new FileWriter("D:\\JSONOutput\\output.json");
```

Comment:
Shouldn't this be a linux file path like, \home\lvuser, since the roboRIO runs a[ linux os](https://www.chiefdelphi.com/t/what-is-the-roborios-os/145971)?

Reply:
probably, was doing a windows output to test if i could "simulate" the code on my pc but didnt work, we can definitely change that

Reply:
changed to "linux file directory change later" 🤷‍♂️